common.lic

```diff
@@ -391,42 +391,50 @@ module DRC
     g
   end
 
-  def play_song?(settings, song_list, worn = true, skip_clean = false)
+  def play_song?(settings, song_list, worn = true, skip_clean = false, climbing = false)
     instrument = worn ? settings.worn_instrument : settings.instrument
     UserVars.song = song_list.first.first unless UserVars.song
+    UserVars.climbing_song = @song_list.first.first unless UserVars.climbing_song || !climbing
```

Comment:
❓ Should this be `song_list` and not `@song_list`?

Reply:
Thank you.  Yes, that variable being wrong causes an extra cycle through the play_song to get that set